xbmc/music/windows/GUIWindowMusicBase.cpp

```diff
@@ -519,6 +522,7 @@ void CGUIWindowMusicBase::GetContextButtons(int itemNumber, CContextButtons &but
       if (item->CanQueue() && !item->IsAddonsPath() && !item->IsScript())
```

Comment:
have not tested this PR yet, but I wonder if `!item->IsAddonsPath()` is still valid here? We have music add-ons that could also benefit from this, like the googleMusic add-ons. @tadly mind testing this with your gmusic add-on while testing the PR?

Reply:
Sure, will do

Reply:
Yes I think it remains a valid test. It is excluding addons themselves, not items that happen to have been added to the item list by an addon.